src/app/action-items/action-items.component.ts

```diff
@@ -20,10 +26,14 @@ export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
   isLoadingActionItems = false;
   isLoadingActionItem = false;
-  actionItemId: string | undefined;
-  actionItem: ActionItem;
-  errorMessage: string | undefined;
-  justAddedItemId: string;
+  actionItemId?: string;
+  actionItem?: ActionItem;
+  errorMessage?: string;
+  actionItemIdForScroll: string;
+  scrollToActionItem: boolean;
+  createDialog: boolean;
```

Comment:
**FIX:** This variable is not used besides the opening dialog. You could safely remove it and pass there proper mode.